web/api/mocks/service/JobsService.js

```diff
@@ -7,12 +7,47 @@ let Error = require('./ErrorService');
  * ....
  *
  * jobId String Id of Job to be fetched
- * returns flow
+ * returns job
  **/
 exports.getJob = function(jobId) {
   let resp;
   if (jobId) {
-    resp = Storage.get('jobs', jobId);
+    resp = Storage.get('job-detail', jobId);
```

Comment:
This should remain "jobs" and the use of job-detail should be removed across all code.